packages/flutter/lib/src/material/ink_sparkle.dart

```diff
@@ -201,6 +201,13 @@ class InkSparkle extends InteractiveInkFeature {
 
     // Creates an element of randomness so that ink emanating from the same
     // pixel have slightly different rings and sparkles.
+    assert((){
+      // In debug builds, randomness can cause flakes. So if a seed has not
+      // already been specified (i.e. for the purpose of the test), set it to
+      // the constant turbulence seed.
+      turbulenceSeed ??= _InkSparkleFactory.constantSeed;
```

Comment:
If the seed is set, it is not random anyways so I let it be. I don't want to break any existing tests or customers.